apis/apis-overview/cloud-cost-api/cloud-cost-trends-api.md

```diff
@@ -0,0 +1,92 @@
+{% endswagger-parameter %}
+
+{% swagger-parameter in="path" name="names" required="true" type="string" %}
+Determines order sequence of queried items via comma-separated list. Dependent on the value of `aggregateBy` to list items. See more [below](cloud-cost-trends-api.md#using-the-names-parameter).
+{% endswagger-parameter %}
+
+{% swagger-parameter in="path" name="AggregateBy" type="string" required="false" %}
+Field by which to aggregate the results. Accepts: `invoiceEntityID`, `accountID`, `provider`, `service`, and `label:<name>`. Supports multi-aggregation using comma-separated lists, such as `aggregate=accountID,service`. Defaults to `service`.
+{% endswagger-parameter %}
+
+{% swagger-parameter in="path" name="accumulate" type="boolean" required="false" %}
+When set to `false`, returns daily time series data vs. cumulative data. Default is `true`.
+{% endswagger-parameter %}
+
+{% swagger-parameter in="path" name="CostMetricName" type="string" required="false" %}
+Determines which cloud cost metric type will be returned. Acceptable values are `AmortizedNetCost`, `InvoicedCost`, `ListCost`, and `NetCost`. Default is `AmortizedNetCost`.
+{% endswagger-parameter %}
+
+{% swagger-parameter in="path" name="filter" type="string" required="false" %}
+Filter your results by a particular category or value. For example, when to only see trends in AWS spend, set this parameter to `filter=provider:"AWS"`. Supports [Filter Parameters (v2)](https://docs.kubecost.com/apis/apis-overview/filters-api) language.
+{% endswagger-parameter %}
+
+{% swagger-response status="200: OK" description="" %}
+<pre class="language-json"><code class="lang-json"><strong>{
+</strong>    "code": 200,
+    "data": {
+        "trends": {
+            "": {
+                "isInfinite": false,
+                "isNaN": false,
+                "value": 0.000
+            }
+        },
+        "window": {
+            "start": "",
+            "end": ""
+        },
+        "comparisonWindow": {
+            "start": "",
+            "end": ""
+        }
+    }
+}
+</code></pre>
+{% endswagger-response %}
+{% endswagger %}
+
+## Calculating trend value
+
+The Trends API determines changes in resource cost usage over time based on the interval set `window` parameter and provides that information via the schema field `value`. Cost usage for the current `window` sampled will be compared with `comparisonWindow`, the window directly before the current window of the same size interval. For example, for `window=3d`, Kubecost will output cost usage for the past three days compared to cost usage of the three days before the start of the window. This means a total of six days of cloud cost data are sampled in order to provide an accurate value.
+
+The equation for calculating `value` is: `value=window/comparisonWindow - 1`
+
+Receiving a positive `value` means your `window` has increased compared to `comparisonWindow`. A negative `value` means spending has decreased.
```

Comment:
Should this be something like "Receiving a positive `value` means your spending has increased in `window` compared to `comparisonWindow` ..." for clarity? As it is reads kind of strange to me